includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -13323,6 +13323,9 @@ class AMP_Allowed_Tags_Generated {
 		'amp-fx' => array(
 			'value_regex_casei' => '(fade-in|fade-in-scroll|fly-in-bottom|fly-in-left|fly-in-right|fly-in-top|parallax)(\\s|fade-in|fade-in-scroll|fly-in-bottom|fly-in-left|fly-in-right|fly-in-top|parallax)*',
 		),
+		'animate-in' => array(),
+		'animate-in-delay' => array(),
+		'animate-in-duration' => array(),
```

Comment:
This manual editing of the generated tags shouldn't be required. These should be automatically populated from `bin/amphtml-update.sh`. If not, then we have a bug to fix in `bin/amphtml-update.py`.

Reply:
I'm woking on this in https://github.com/Automattic/amp-wp/pull/1386. I can see there is indeed a problem with how the script handles `$REFERENCE_POINTS` in the spec.

Reply:
In the mean time, we can ignore this and merge this as-is.